lectures/L11-slides.tex

```diff
@@ -385,7 +385,7 @@
 
 \end{verbatim}
 
-To assign an entry: \texttt{courseMonths[0] = September;}
+To assign an entry: \texttt{courseMonths[0] = Months.September;}
```

Comment:
I don't think you have to do this in C++; at least when I was testing this out it was much happier if I didn't use Months.